graphite/graphite.go

```diff
@@ -2,6 +2,7 @@
 package graphite // import "bosun.org/graphite"
 
 import (
+	"bosun.org/cache"
```

Comment:
Actually, this package should not depend on cache at all. Caching should be at a higher level.